src/components/PhotoGallery/PhotoDetails.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component, PropTypes } from 'react';
+import { Link } from 'react-router';
+import { get } from '../../api/request';
+
+export default class PhotoDetails extends Component {
```

Comment:
Instead of this component having it's own state, lets connect and save the photo info to the redux store.